dhctl/pkg/terraform/runner.go

```diff
@@ -513,17 +513,31 @@ func (r *Runner) Destroy() error {
 		return nil
 	}
 
+	err := r.runBeforeActionAndWaitReady()
```

Comment:
why did we move runBeforeActionAndWaitReady before autoapprove confirmation?

Reply:
fixed, moved runBeforeActionAndWaitReady after "delete objects from cloud" approve